### PR TITLE
Docker: Add script to update docker containers

### PR DESCRIPTION
**Pull request by Haroon-Khel**
- [x] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

ref https://github.com/adoptium/infrastructure/issues/2070

Makes https://github.com/adoptium/infrastructure/pull/2398 redundant as I could not get cron jobs to work in the containers, so have decided to implement a cron job on the host

### CODE REVIEW

**Review by karianna — COMMENTED** [review]
Is this a wanted change?  Do we not need to pin our versions for reproducible builds?

**Comment by sxa** [comment]
> Is this a wanted change? Do we not need to pin our versions for reproducible builds?

That can be a separate discussion but I do not want machines on the network that are unsupported or do not undergo security patching. This brings the static docker containers up to the same patching that we have for all our other machines as per the playbooks.

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/scripts/updatepackages.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    elif [[ $OS =~ "Alpine" ]]; then
+        installCommand="apk update && apk upgrade"
+    else 
+        echo "Unrecognised OS, skipping package update"
+        continue
+    fi
+    echo "Updating packages for container $container"
+    docker exec -it $container sh -c "$installCommand"
+    echo "=============================================="
+done
```
Comment (sxa) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/scripts/updatepackages.sh:22]: Should have a newline at the end

**Review by sxa — CHANGES_REQUESTED** [review]
One minor point, otherwise I believe this is good.

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/scripts/updatepackages.sh (hunk 1 above)
Comment (Haroon-Khel) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/scripts/updatepackages.sh]: Done

**Review by sxa — APPROVED** [review]

**Pull request merged**